recipes/mpdecimal/2.5.x/conanfile.py

```diff
@@ -205,7 +206,7 @@ def package(self):
             mpdecppdir = self.build_path / "libmpdec++"
             copy(self, "mpdecimal.h", src=mpdecdir, dst=pkg_dir / "include")
             if self.options.cxx:
-                copy(self, "decimal.hh", src=mpdecppdir, dst=pkg_dir / "include")
+                copy(self, "decimal.hh", src=self.source_path / "libmpdec++", dst=pkg_dir / "include")
```

Comment:
This section is weird. For some reason, in the cache folder, most of the the files are symlinks in 4.0.0, but actual files in 2.5.1. So we need to copy from the source folder instead. But this only affects the `libmpdec++` folder. In fact, making this change to the `libmpdec` folder causes the C header to not get packaged. I don't understand any of why this is the case, but it works like this.

Also to note: I'm just being consistent here with the usage of `*_path` variables, I know they're deprecated.

Reply:
I haven't tested yet, but I'm hoping the weirdness will go away in the followup cleanup PR.